backoffice/templates/backoffice/vehicles/vehicles_list_item.html

```diff
@@ -10,7 +10,7 @@
     </div>
   </td>
   <td class="px-6 py-3 text-sm font-medium text-gray-500">
-    {{ v.registration }}
+    <a href="{% url "vehicle_details" v.id %}" class="underline">{{ v.registration }}</a>
```

Comment:
I'm not sure about the hard underline here - I don't think it looks quite right. Maybe a underline on hover instead?